src/components/dropdown/Dropdown.tsx

```diff
@@ -195,6 +195,7 @@ export class Dropdown<T = LabelValue> extends React.Component<
               ...base, maxHeight: `${maxHeight}px`
             }),
             input: (base: CSSProperties) => ({...base, margin: (size === 'small') ? '0 2px' : undefined, color: 'inherit'}),
+            multiValue: (base: CSSProperties) => ({ ...base, margin: '0' })
```

Comment:
what is this used to ?

Reply:
It is to remove the unwanted margin around the token render

It is now handled by the field, here :) https://github.com/SymphonyPlatformSolutions/symphony-bdk-ui-toolkit-styles/pull/194/files#diff-77fa5ebeb624b65afaf890995a095d3c9e784df4f689f186b8f871a0059f7406R184